bittensor/chain_data.py

```diff
@@ -659,7 +659,7 @@ class PrometheusInfo:
 
     block: int
     version: int
-    ip: str
+    ip: str  # string representation of converted ip to int using tool like https://tools.iplocation.net/ip-to-integer-converter
```

Comment:
I would replace the link to this one

https://netaddr.readthedocs.io/en/latest/api.html#netaddr.IPAddress.value

We actually use netaddr to do this in the networking.py file. This is their API documentation.

Reply:
Done